lib/common/utils.js

```diff
@@ -254,7 +254,7 @@ function cleanChildrenRoutes(routes, isChild = false) {
   return routes
 }
 
-exports.createRoutes = function createRoutes(files, srcDir, pagesDir) {
+exports.createRoutes = function createRoutes(files, srcDir, pagesDir, creationOptions) {
```

Comment:
Name to be `options` or use `destructuring assignment` should be more appropriate.

Reply:
Now I use there a destructuring assignment as you recomend, but I don't know if I do it fine, it's the first time that I use it as function arguments